lib/factory-helper/internet.rb

```diff
@@ -104,13 +104,28 @@ def url(host = domain_name, path = "/#{user_name}")
       end
 
       def slug(words = nil, glue = nil)
-        glue ||= %w[- _ .].sample
+        glue ||= %w[- _ .].sample(:random => FactoryHelper::Config.random)
         (words || FactoryHelper::Lorem::words(2).join(' ')).gsub(' ', glue).downcase
       end
 
       def device_token
-        rand(16 ** 64).to_s(16).rjust(64, '0').chars.to_a.shuffle.join
+        FactoryHelper::Config.random.rand(16 ** 64).to_s(16).rjust(64, '0').
+          chars.to_a.shuffle(:random => FactoryHelper::Config.random).join
       end
+
+      private
```

Comment:
Didn't realize it was documented as part of the public API. Can't imagine people using it, but hey...
An alternative is to give it default args, which seems like a silly idea, but doesn't break backwards compatibility.